packages/core/src/storybook/stand-alone-documentaion/welcome/welcome.mdx

```diff
@@ -40,3 +41,5 @@ Read our<Link href="https://github.com/mondaycom/vibe/blob/master/CONTRIBUTING.m
 <Contributors />
 
 <OtherContributorsList />
+
+<DocFooter feedbackFormLink="https://forms.monday.com/forms/213ebddcb0d423ae5b6178fb6e8f7b3d?r=use1" />
```

Comment:
I removed it from all pages because it was a bit weird to have a feedback section in the migration guide and another this one as well. I guess I'll revert it for now and we'll think about it because I don't like it either

Reply:
agree